components/MainHeader/MainHeader.tsx

```diff
@@ -0,0 +1,34 @@
+import Link from 'next/link';
+
+import styles from './mainHeader.module.scss';
+
+const links = [
+  { label: 'Aktualności', href: '/aktualnosci', openInNewWindow: false, icon: 'inbox' },
+  { label: 'O serwisie', href: '/o-serwisie', openInNewWindow: false, icon: 'question-circle' },
+  { label: 'Zgłoś uwagi', href: '/zglos-uwagi', openInNewWindow: false, icon: 'comment' },
+  {
+    label: 'Facebook',
+    href: 'https://facebook.com/polskifrontend',
+    openInNewWindow: true,
+    icon: 'facebook',
+  },
+];
+
+export const MainHeader = () => {
+  return (
+    <header>
+      <ul className={styles.headerList}>
+        {links.map(({ label, href, openInNewWindow, icon }) => (
+          <li className={styles.headerItem} key={href}>
+            <Link href={href}>
```

Comment:
Next zaleca dodawać passHref do linków które w środku mają `a`

https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag

```suggestion
            <Link href={href} passHref>
```

Reply:
Tu jest napisane coś wprost odwrotnego @AdamSiekierski :P

Reply:
Sorry, rzeczywiście. Nawyk ze styled-components :)